CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -502,20 +461,24 @@ public function processCreditCard($submittedValues) {
     $paymentParams['contactID'] = $this->_contactId;
     CRM_Core_Payment_Form::mapParams($this->_bltID, $this->_params, $paymentParams, TRUE);
 
+    // add some financial type details (as contributionType_name) to the params list
```

Comment:
Because it's used in few Payment processor, as per ```grep```
CRM/Core/Payment/Elavon.php:105:     *  $params['contributionType_name'  ];
CRM/Core/Payment/FirstData.php:137:    //  $requestFields[       ''  ]          =  $params[ 'contributionType_name'  ];
CRM/Core/Payment/Realex.php:369:    $this->_setParam('varref', $params['contributionType_name']);